src/main/app/angular.json

```diff
@@ -69,12 +69,12 @@
                 {
                   "type": "initial",
                   "maximumWarning": "2mb",
-                  "maximumError": "5mb"
+                  "maximumError": "6mb"
                 },
                 {
                   "type": "anyComponentStyle",
                   "maximumWarning": "6kb",
-                  "maximumError": "10kb"
+                  "maximumError": "150kb"
```

Comment:
I assume as well the Formio needs this. So far this code base managed quite well in very lean components. Benefitting performance.

But not really an issue. Ofc for good performance you don't want loads of very large components. We could even argue to put the warning to 20 or 30k. Since both are very far off now. But ofc good to keep and eye on the warnings (anyways).

Reply:
The increase of 10kb to 150 kb is needed beacuse of the things I added to package.json.

The error you get with 10k is:
Error: /home/andy/IdeaProjects/dimpact-zaakafhandelcomponent/src/main/app/src/app/formulieren/formio-wrapper/formio-wrapper.component.less exceeded maximum budget. Budget 10.00 kB was not met by 132.64 kB with a total of 142.64 kB.

But I think it is possible to dynamically load the Formio Angular component the moment is is needed.

Reply:
Quite a lot of kB's not met 😉 .  Angular supports separate budgets for lazy-loaded bundles; so another reason to get our code base chunked to at least two pieces. User and ZAPS.